pkgs/by-name/ne/neomutt/package.nix

```diff
@@ -33,6 +33,7 @@
   enableLua ? false,
   enableSmimeKeys ? true,
   withContrib ? true,
+  enableNotmuch ? true,
```

Comment:
Nit. Would withNotmuch be more appropriate here?

Reply:
Yeah, I prefer the `with` prefix too, but as there was `enableLua` and `enableSmimekeys` I pick enable, but I'll switch to `with`.